### PR TITLE
Added TestHostLaunched event to DataCollector docs

### PR DESCRIPTION
**Pull request by karanjitsingh**
15.7 blame data collector has this feature and `Microsoft.Testplatform.ObjectModel` as well.

### CODE REVIEW

File: docs/extensions/datacollector.md [hunk 1]
```diff
@@ -93,12 +93,14 @@ DataCollectors can choose to subscribe to any of the four events exposed by `Dat
 2. TestSessionEnd : Raised when test execution session ends.
```
Comment (smadala) [inline docs/extensions/datacollector.md]: five events.

File: docs/extensions/datacollector.md [hunk 2]
```diff
@@ -93,12 +93,14 @@ DataCollectors can choose to subscribe to any of the four events exposed by `Dat
...
 3. TestCaseStart : Raised when test case execution starts.
 4. TestCaseEnd : Raised when test case execution ends.
+5. TestHostInitialized : Raised when test host process has been initialized.
```
Comment (smadala) [inline docs/extensions/datacollector.md]: Write a note saying that this event available from vstest 15.7.

Comment (pvlakshm) [inline docs/extensions/datacollector.md]: From which version onwards will this be available? Can you mention that, please.

Comment (smadala) [inline docs/extensions/datacollector.md]: Change this to `TestHostLaunched`.

Comment (acesiddhu) [reply docs/extensions/datacollector.md]: 15.7 blame data collector has this feature and Microsoft.Testplatform.ObjectModel this as well.

File: docs/extensions/datacollector.md (hunk 1 above)
Comment (karanjitsingh) [reply docs/extensions/datacollector.md]: updated

**Pull request merged**